src/workloads/sharding/WriteOneWithoutShardKeyReplicaSet.yml

```diff
@@ -0,0 +1,76 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  Run updateOnes, deleteOnes, and findAndModifys without shard key or id in their queries against a
+  replica set.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+  Database: &Database test
+  DocumentCount: &DocumentCount 10000  # Number of documents to insert and modify.
+
+Actors:
+- Name: LoadInitialData
+  Type: MonotonicSingleLoader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    Database: *Database
+    Document: {x: 1, y: 1, z: 1}
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: CRUDOpsWithoutShardKey
+  Type: CrudActor
+  Threads: 1  # Use 1 thread to avoid updates throwing WriteConflict errors.
+  Phases:
+  - *Nop
+  - Repeat: *DocumentCount
+    Database: *Database
+    Collection: Collection0
+    Operations:
+    - OperationMetricsName: UpdateOneWithoutShardKey
```

Comment:
There's no concept of a shardKey so truly this is just updates, deletes, and findAndModify testing, we should change the names of the operation metrics to reflect that.

That being said, if we already have this for replica sets, we might not need this genny workload. However, now that it's written, it might be easier to compare timing between this and the other two workloads since it's modifying the same number of documents per workload.

Reply:
removed and renamed file to remove notion of shard key. For the second comment I dont think our previously written tests are against replica sets (specifically) if thats what you meant? I wanted a test case that mirrored our other ones just performed on a replica set.

Reply:
Not the test we wrote previously for this project, but just genny tests that already exist. But yeah, it's nice to have ones that directly mirror our other tests so I don't mind keeping this workload.